### ISSUE TITLE
Bakups fail behind htaccess auth

### ISSUE THREAD

**Issue by owaincuvelier** [humanmade/backupwordpress#782]
Backups begin working again when htaccess auth is removed.

https://app.intercom.io/a/apps/7f1l4qyq/inbox/all/conversations/802616820

**Comment by pdewouters** [comment]
@rmmcue I'm thinking Backdrop would have issues doing a `wp_remote_post` in this case?

**Comment by willmot** [comment]
Yeah it would, we could detect this and ask them to enter their username and password for basic auth.

**Comment by rmccue** [comment]
Correct, will also cause cron/etc to fail. Should just tell the user it's not supported, IMO, as it'll likely break other parts of WP.

**Comment by pdewouters** [comment]
Right, we use Backdrop for calculating the folder sizes too, so even if I reverted - it would still cause issues.

**Comment by pdewouters** [comment]
@owaincuvelier you can suggest that the user installs the Members plugin and sets it to private blog.
https://wordpress.org/plugins/members/

**Comment by pdewouters** [comment]
Still on the fence about this.This user has a point https://wordpress.org/support/topic/314-is-only-one-that-works-for-me?replies=11#post-6767727

**Comment by willmot** [comment]
I guess the user will already get our wp-cron not accessible warning in this instance?

**Comment by willmot** [comment]
I think we should add a check for this (as we do for wp-cron) and warn the user 👍

We probably want to skip the cron check if the above warning is triggered as no point showing both.
